invokeai/app/invocations/flux_denoise.py

```diff
@@ -65,9 +63,9 @@
     title="FLUX Denoise",
     tags=["image", "flux"],
     category="image",
-    version="3.3.0",
+    version="3.3.1",
```

Comment:
Removing `WithMetadata` and `WithBoard` is a breaking change to the node because it removes fields. Let's bump this to v4.

Reply:
Updated.